phoenix-core/src/main/java/org/apache/phoenix/schema/types/PDouble.java

```diff
@@ -17,15 +17,16 @@
  */
 package org.apache.phoenix.schema.types;
 
-import com.google.common.base.Preconditions;
-import com.google.common.primitives.Doubles;
```

Comment:
Should I combine all the commits into a single one?
That is one issue for one commit.